tests/integration/test_charm.py

```diff
@@ -41,6 +34,14 @@ async def test_build_and_deploy(ops_test: OpsTest):
     )
     assert ops_test.model.applications[APP_NAME].units[0].workload_status == "active"
 
+
+async def test_seldon_deployment(ops_test: OpsTest):
+    namespace = ops_test.model_name
+    client = Client()
+
+    this_ns = client.get(res=Namespace, name=namespace)
+    this_ns.metadata.labels.update({"serving.kubeflow.org/inferenceservice": "enabled"})
```

Comment:
I don't think this will set the label, it will just update the in-memory Python object

Reply:
Good catch, there was a missing call to `client.patch()`